R/calculate_tdm.R

```diff
@@ -0,0 +1,305 @@
+#' Calculate and add the transition disruption metric to a dataset
+#'
+#' This function takes a typical PACTA output, and uses it to calculate the
+#' transition disruption metric.
+#'
+#' @param data A dataset, like the `*_results_portfolio.rda` outputs of PACTA.
```

Comment:
maybe this should say that it accepts a data.frame (or a tibble) explicitly, because it almost sounds like it might take a path to an RDA file (which is actually a RDS file).

Reply:
Yup good shout! Normally I would point to a dummy dataset but we still need to create those.